helmfile.d/values/falco/falco-common.yaml.gotmpl

```
@@ -136,11 +136,16 @@ customRules:
     # This will be added in a later falco rules version as well
     # The fix was added upstream here (with a new condition): https://github.com/falcosecurity/rules/pull/177
     - macro: allowed_clear_log_files
-      condition: (
+      condition: or (
         proc.name=containerd and (
           fd.name startswith "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/" or
           fd.name startswith "/var/lib/containerd/tmpmounts/"
+        )) or (
+        proc.name=nvidia-installe and (
+          fd.name startswith "/var/log/nvidia-installer.log"
         ))
```

```suggestion
      condition: or (
          proc.name = containerd and (
            fd.name startswith "/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots/" or
            fd.name startswith "/var/lib/containerd/tmpmounts/"
          )
        ) or (
          proc.name = nvidia-installe and
          fd.name startswith "/var/log/nvidia-installer.log"
        )
```
Nit for improved readability